docs-ref-conceptual/authenticate-azure-cli-interactively.md

```diff
@@ -13,11 +13,79 @@ ms.custom: devx-track-azurecli
 
 # Sign in interactively with Azure CLI 
 
-Interactive logins to Azure offer a more intuitive and flexible user experience. Interactive login with Azure CLI allows users to authenticate to Azure directly through the [az login](/cli/azure/reference-index#az-login) command, which is useful for ad-hoc management tasks and for environments that require manual sign-in, such as those customers with multi-factor authentication (MFA). This method simplifies access for script testing, learning, and on-the-fly management without needing to preconfigure service principals or other noninteractive authentication methods.
+Interactive logins to Azure offer a more intuitive and flexible user experience. Interactive login
+with Azure CLI allows users to authenticate to Azure directly through the [az login](/cli/azure/reference-index#az-login)
+command, which is useful for ad-hoc management tasks and for environments that require manual
+sign-in, such as those customers with multi-factor authentication (MFA). This method simplifies
+access for script testing, learning, and on-the-fly management without needing to preconfigure
+service principals or other noninteractive authentication methods.
```

Comment:
Any reason for breaking these lines? They will make `diff`ing difficult if the position of line break character `\n` changes.

Reply:
That is a feature of vsCode and used extensively by both PowerShell and Azure PowerShell writers -- Ctrl D causes every line to not exceed 100 (?) characters.